core/src/avm2/globals/flash/filters/DropShadowFilter.as

```diff
@@ -0,0 +1,33 @@
+package flash.filters {
+	public final class DropShadowFilter extends BitmapFilter {
+		public var alpha: Number;
+		public var angle: Number;
+		public var blurX: Number;
+		public var blurY: Number;
+		public var color: uint;
+		public var distance: Number;
+		public var hideObject: Number;
+		public var inner: Boolean;
+		public var knockout: Boolean;
+		public var quality: int;
+		public var strength: Number;
+
+		public function DropShadowFilter(distance:Number = 4.0, angle:Number = 45, color:uint = 0, alpha:Number = 1.0, blurX:Number = 4.0, blurY:Number = 4.0, strength:Number = 1.0, quality:int = 1, inner:Boolean = false, knockout:Boolean = false, hideObject:Boolean = false) {
```

Comment:
Hello and welcome back! The ActionScript linter seems to think this line is too long. Can you split this declaration into a few lines? Thanks!